src/engine-actions/close-channels.js

```diff
@@ -0,0 +1,70 @@
+const {
+  listChannels,
+  closeChannel
+} = require('../lnd-actions')
+/**
+ * Closes a channel given the channel point
+ *
+ * @param {Boolean} [options.outbound=true] outbound is true if checking outbound channels, false if inbound
+ * @returns {Promise<Array.<Object>>}
+ */
+async function closeChannels ({ force = false } = {}) {
+  const { channels = [] } = await listChannels({ client: this.client })
```

Comment:
If channels dont exist, we should log and return immediately. If we don't, we'll always receive `Successfully closed channels` even if there weren't any, which might be confusing